mlir/lib/CAPI/Target/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_mlir_upstream_c_api_library(MLIRCAPITarget
+  LLVMIR.cpp
+
+  LINK_LIBS PUBLIC
+  MLIRToLLVMIRTranslationRegistration
+  MLIRCAPIIR
+  MLIRLLVMToLLVMIRTranslation
+  MLIRSupport
```

Comment:
I suspect we may want to link the relevant LLVM target for LLVM C APIs.

Reply:
do you know which one is? i'm a bit lost here